src/api/services/judge0.ts

```diff
@@ -33,14 +33,18 @@ class JudgeServer {
       payload
     );
   }
-
   getSubmission(token: string, base64: boolean): Promise<AxiosResponse> {
     return this.axiosInstance.get(
       `/submissions/${token}?base64_encoded=${base64}`
     );
   }
+  getSubmissionVariant({ token, base64 }): Promise<AxiosResponse> {
```

Comment:
It might be helpful to leave a comment explaining why we have this function as well as the previous function